pkgs/by-name/sp/spiel/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  python312Packages,
+  callPackage,
+  fetchPypi,
+}:
+
+python312Packages.buildPythonApplication rec {
+  pname = "spiel";
+  version = "0.5.1";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
```

```suggestion
    pname = "spiel";
    inherit version;
```
`pname` shouldn't be interpolated.